src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/jpa/NextReviewDate.kt

```diff
@@ -0,0 +1,24 @@
+package uk.gov.justice.digital.hmpps.incentivesapi.jpa
+
+import org.springframework.beans.factory.annotation.Value
+import org.springframework.data.annotation.Id
+import org.springframework.data.annotation.Transient
+import org.springframework.data.domain.Persistable
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+data class NextReviewDate(
+  @Id
+  val bookingId: Long,
```

Comment:
Not sure it would help much and it would be another thing to be sure we keep up-to-date/in-sync with Prison API which I'd personally avoid.

It would be more useful to add support for the Offender Search `POST` endpoint that gets offender by bookingIds as I could avoid few requests here and there, but that has slightly different role requirements so I didn't want to add it as part of this.